docs/design/assets/release.uml

```diff
@@ -0,0 +1,69 @@
+@startuml
+title Release Workflow
+autonumber
+actor RM order 10
+actor Reviewer order 20
+database Upstream order 30
+control CI order 40
+participant Publish order 50
+database Quay order 60
+
+activate RM
+RM -> RM: make initiate-release
+RM -> Upstream: create pull-request
+deactivate RM
+
+activate Upstream
+Upstream -> CI: run checks, e2e
+CI -[#green]> Upstream:  pass
+deactivate Upstream
+
+actor Reviewer
+Reviewer -> Upstream: approve & merge
+
+== candiate release prep ==
+activate CI
+Upstream -> CI: release workflow (release.yaml)
+CI -> CI: run e2e
+CI -> CI: generate release notes
+CI -> CI: create GH release notes
+CI -> Upstream: git tag release and git push tag
+CI -> Upstream: publish pre release to GH
+deactivate CI
+== candiate release prep ==
+
+== candiate release  ==
+Upstream -> CI: start candidate release (olm-candidate.yaml)
+CI -> Publish: candidate release (olm-publish.yaml)
+CI -> Publish: run olm-publish (channels: development, candidate)
+    activate Publish #FFBBBB
```

Comment:
Non-blocking question the difference in colour between this and the  #DarkSalmon on line 62 is supposed to differentiate the candidate and stable versions, correct?

Reply:
Thats correct :-) and that it doesn't show the entire flow again

Reply:
Done , and also noticed that the `candidate` was spelt wrong.